integration-cli/docker_cli_daemon_test.go

```diff
@@ -1723,3 +1723,30 @@ func (s *DockerDaemonSuite) TestDaemonStartWithoutHost(c *check.C) {
 	}()
 	c.Assert(s.d.Start(), check.IsNil)
 }
+
+func (s *DockerDaemonSuite) TestBridgeIPIsExcludedFromAllocatorPool(c *check.C) {
+	defaultNetworkBridge := "docker0"
+	deleteInterface(c, defaultNetworkBridge)
+
+	bridgeIP := "192.169.1.1"
+	bridgeRange := bridgeIP + "/30"
+
+	err := s.d.StartWithBusybox("--bip", bridgeRange)
+	c.Assert(err, check.IsNil)
+	defer s.d.Restart()
+
+	var cont int
+	for {
```

Comment:
with a /30, the exhaustion will happen for the 2nd container.
Thats because, /30 will consume 1 for subnet (192.169.1.0) and 1 for broadcast (192.169.1.3).
With 192.169.1.1 taken for bip, the only other ip for the container will be the 192.169.1.2

Do you really need a for loop for that ? Unless you change the mask to < 30